Tests/SuperLinq.Test/FallbackIfEmptyTest.cs

```diff
@@ -7,14 +7,58 @@ public void FallbackIfEmptyWithEmptySequence()
 	{
 		using var source = Enumerable.Empty<int>().AsTestingSequence(maxEnumerations: 2);
 		source.FallbackIfEmpty(12).AssertSequenceEqual(12);
-		source.FallbackIfEmpty(12, 23).AssertSequenceEqual(12, 23);
+	}
+
+	[Fact]
+	public void FallbackIfEmptyWithCollectionSequence()
+	{
+		using var source = Enumerable.Empty<int>().AsTestingCollection(maxEnumerations: 2);
```

Comment:
`maxEnumerations` does not need to be set to `2`, now that we are not executing `FallbackIfEmpty()` twice.